lib/plugins/aws/provider/awsProvider.test.js

```diff
@@ -398,20 +439,56 @@ describe('AwsProvider', () => {
           },
         }));
 
-      return awsProvider.getServerlessDeploymentBucketName(options.stage, options.region)
+      return awsProvider.getServerlessDeploymentBucketName()
         .then((bucketName) => {
           expect(describeStackResourcesStub.called).to.be.equal(false);
           expect(bucketName).to.equal('custom-bucket');
           awsProvider.request.restore();
         });
     });
-  });
 
-  describe('#getStackName', () => {
-    it('should return the stack name', () => {
-      serverless.service.service = 'myservice';
+    describe('#getStage()', () => {
+      it('should prefer options over config or provider', () => {
```

Comment:
I know that this might be pedantic (:grimacing: but could we please add a space here (so that all the test have the same look and feel?